uwsgi/public.wsgi

```diff
@@ -37,6 +37,8 @@ cache.make_cache("blob", 500, 3600)
 cache.make_cache("blob_schema", 50, 24 * 60 * 60)
 cache.make_cache("blob_version", 500, 60)
 
+cache.make_cache("rules", 20, 30)
```

Comment:
Yah, cache more! :) Is the product+build target in the request URLs? I can do some cli log parsing to give you a good idea of how many unique combos there are.